demos/iot_config.h

```diff
@@ -34,6 +34,8 @@
 #define IOT_DEMO_ROOT_CA               "" /* Command line: -r */
 #define IOT_DEMO_CLIENT_CERT           "" /* Command line: -c */
 #define IOT_DEMO_PRIVATE_KEY           "" /* Command line: -k */
+#define IOT_DEMO_USER_NAME             "" /* Command line: -m */
+#define IOT_DEMO_PASSWORD              "" /* Command line: -w */
```

Comment:
If the demo config defines these macros, should `tests/iot_config.h` also be changed to define `IOT_TEST_USER_NAME` and `IOT_TEST_PASSWORD`?

Reply:
Good catch. Fixed.